### PR TITLE
Use iso3 2-letter code in address

### PR DESCRIPTION
**Pull request by lacazeto**
- Removed the option `show_destinationCountryCode` as it was showing the country info in a weird place, as a 3-letter code and would mostly just cause confusion along with this ticket's feature request. 

- Added the 2-letter country code at the last line of the address we show in the `Address` component. This is no longer optional and it will always appear, unless we don't recognize the value coming from the backend, then nothing will be rendered for the country info.

### CODE REVIEW

**Review by Kafoozelum — APPROVED** [review]

File: src/js/lib/helpers.js [hunk 1]
```diff
@@ -47,3 +47,260 @@ exports.checkUrl = function (url) {
   }
   return true
 }
+
+exports.iso3CountryCode3to2LetterConversor = function (countryCode) {
```
Comment (palak-singhal) [inline src/js/lib/helpers.js:51]: a small typo:-  `iso3CountryCode3to2LetterConverter` or is it intentionally? 😅

**Review by palak-singhal — CHANGES_REQUESTED** [review]
a small typo, otherwise looks good

File: src/js/lib/helpers.js (hunk 1 above)
Comment (lacazeto) [reply src/js/lib/helpers.js:51]: No, not intentional! Maybe a bit internationalised with the name in two languages 😅

**Review by palak-singhal — APPROVED** [review]
LGTM

**Pull request merged**